setup.py

```diff
@@ -26,8 +26,8 @@
     if py_version < (3, 3) and not is_pypy: # PyPy3 masquerades as Python 3.2...
         raise RuntimeError('On Python 3, Pyramid requires Python 3.3 or better')
 else:
-    if py_version < (2, 6):
-        raise RuntimeError('On Python 2, Pyramid requires Python 2.6 or better')
+    if py_version < (2, 7):
+        raise RuntimeError('On Python 2, Pyramid requires Python 2.7 or better')
```

Comment:
Let's say we introduce set/dict comprehensions for example, and don't catch it because we no longer test on Py 2.6 but still allow people to install and attempt to run on Py 2.6, this would break things. I'd rather have a clear cut off.

If you'd prefer to have a soft cut-off, I will happily make the change, I am not dead set on it one way or the other, but we have the protection at the moment and it seemed like updating it would be the right thing to do.